misc/python/materialize/version_list.py

```diff
@@ -49,8 +49,10 @@
 """
 MIN_ANCESTOR_MZ_VERSION_PER_COMMIT: dict[str, MzVersion] = {
     # insert newer commits at the top
+    # PR#23659 (persist-txn: enable in CI with "eager uppers") introduces regressions against v0.79.0
+    "c4f520a57a3046e5074939d2ea345d1c72be7079": MzVersion.parse_mz("v0.80.0"),
```

Comment:
@nrainer-materialize Did I do this right? The version in the value is supposed to be the next version of mz (even though that tag doesn't exist yet)?

Reply:
Yes, assuming that your changes will be included in tomorrow's 0.80.0 release.